test/ProductConstructionService.DependencyFlow.Tests/PullRequestBuilderTests.cs

```diff
@@ -10,6 +10,7 @@
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.VisualStudio.Services.Common;
 using Moq;
+using NetTopologySuite.IO;
```

Comment:
this looks weird, are we using it?
The GH readme says 

> An excerpt from [JTS website](http://sourceforge.net/projects/jts-topo-suite) explains the capabilities of NTS too: "The JTS Topology Suite is an API for modelling and manipulating 2-dimensional linear geometry. It provides numerous geometric predicates and functions. JTS conforms to the Simple Features Specification for SQL published by the Open GIS Consortium.

:O

Reply:
```suggestion
```